src/sql/storage-usage-azure.sql

```diff
@@ -23,30 +21,56 @@ FROM
     FROM master.sys.master_files
     WHERE physical_name LIKE 'https:%'
 ) AS alloc(size_tb)),
+next_gen_gp_stats ([Remaining Next-Gen files])
+AS (
+SELECT
+		[Remaining Next-Gen files] = CAST((32768 - file_count) AS int)
+FROM
+( SELECT COUNT(*)
+    FROM master.sys.master_files
+    WHERE physical_name LIKE 'C:\ManagedDisks%'
+) AS alloc(file_count)),
 volumes as (
-SELECT	Storage = CASE WHEN volume_mount_point = 'http://' THEN 'Remote storage'
-                        ELSE 'Local SSD'
-                    END,
+SELECT	Storage = CASE
```

Comment:
With PaaSV2 and containers, MountPoint for both BC and Next-Gen GP will be C:\, so I had to use additional folder as well to distinguish between them